config/azure/build.yaml

```diff
@@ -64,12 +64,12 @@ jobs:
      displayName: 'Build Windows'
      condition: eq( variables['Agent.OS'], 'Windows_NT' )
 
-   - script: |
-       call "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Auxiliary\Build\vcvarsall.bat" x64
-       scons testCore testCorePython testScene testImage testAlembic testAppleseed OPTIONS=${{ parameters.options }}
-       more test/IECore/results.txt
-     displayName: 'Test'
-     condition: eq( variables['Agent.OS'], 'Windows_NT' )
+  #  - script: |
+  #      call "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Auxiliary\Build\vcvarsall.bat" x64
+  #      scons testCore testCorePython testScene testImage testAlembic testAppleseed OPTIONS=${{ parameters.options }}
+  #      more test/IECore/results.txt
+  #    displayName: 'Test'
+  #    condition: eq( variables['Agent.OS'], 'Windows_NT' )
```

Comment:
If none at all work, can we just delete this task rather than commenting out?

Reply:
Taken care of in 0fdd8a4